Godeps/Godeps.json

```diff
@@ -1,31 +1,71 @@
 {
-	"ImportPath": "k8s.io/spartakus",
-	"GoVersion": "go1.6",
-	"GodepVersion": "v66",
+	"ImportPath": "github.com/kubernetes-incubator/spartakus",
```

Comment:
this should have been updated a little while ago when the project URL was moved from k8s.io/spartakus to github.com/kubernetes-incubator/spartakus